tools/check.rb

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env ruby
+
+require 'yaml'
+require 'json'
+
+HUB_FED_CONFIG_DIRECTORY = File.join(__dir__, '../../verify-hub-federation-config')
+
+ENVIRONMENTS = ["prod", "integration"]
+ENVIRONMENT = ARGV[0]
+ENTITY_ID = ARGV[1]
+MSA = ARGV[2] == '--msa'
+
+if ENVIRONMENT.nil?
+  puts "USAGE: ./check.rb <environment> <entity_id> [--msa optional]"
```

Comment:
I think `[--msa]` is fine without the word 'optional'. However I think a marginally better interface might be:

```
./check.rb [sp|msa] <environment> <entity_id>
```

Because then you have to specify in both cases and it might be slightly less likely to get confusing.

But I'm not fussed if you disagree. This is fantastic - thank you.

Reply:
Feel free to change/do whatever with the script - it's yours :) ... the main logic is there and can be used to do the thing ;)